extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -375,6 +440,14 @@ class AmpLightbox extends AMP.BaseElement {
     this.triggerEvent_(LightboxEvents.CLOSE);
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  isInIframe_() {
+    return this.win != this.win.top;
```

Comment:
this will not work in Search Viewer.

I'm wondering if there is a way to check which window the AMP runtime is installed 
maybe @choumx knows